lib/main_containers.pm

```diff
@@ -131,12 +136,16 @@ sub load_host_tests_containerd_nerdctl {
     loadtest 'containers/containerd_nerdctl';
 }
 
-sub load_host_tests_helm() {
+sub load_host_tests_helm {
+    my ($run_args) = @_;
     if (is_sle('15-sp3+')) {
-        loadtest "containers/helm_eks";
-        loadtest "containers/helm_k3s";
+        $run_args->{k8s_env} = "k3s";
+        load_helm_test($run_args);
```

Comment:
Maybe we can skip `$run_args` for calling the test module. So, something like this:

```
load_helm_test(); # k3s as default if no arguments
load_helm_test(k8s_env => 'eks');
```

Then, in the `run` method of `helm.pm` you can do something like:
```
sub run {
    my ($self, %args) = @_;
    $args{k8s_env} //= 'k3s'; # use k3s as default
    
    my $is_k3s = $args{k8s_env} == 'k3s';
    my $is_eks = $args{k8s_env} == 'eks';
    ....
```

Reply:
I applied a mix solution